README.md

```diff
@@ -269,6 +269,8 @@ Where `wf_options.json` would contain:
 ```
 {
   "jes_gcs_root": "gs://my-bucket/workflows"
+  "account_name": "my.google.account@gmail.com"
+  "refresh_token": "1/Fjf8gfJr5fdfNf9dk26fdn23FDm4x"
```

Comment:
These new ones are optional? Can we indicate that somehow (and say what happens if they are/aren't used)?